app/javascript/react/assets/styles/Typography/Typography.style.tsx

```diff
@@ -39,4 +39,12 @@ const H6 = styled.h6<{ bold?: boolean; uppercase?: boolean }>`
   letterspacing: ${(p) => (p.uppercase ? "0.1px" : "normal")};
 `;
 
-export { H1, H2, H3, H4, H5, H6 };
+const buttonText = css`
```

Comment:
Is it the same as `H4` but with `text-transform: uppercase;`? If so, maybe we can add this as parameter to existing H4?

Reply:
I'm not sure, I think it's nicer to have something dedicated to buttons and not use "headers" for button text. But that's just my intuition, let's see what Kasia will say is more common or sth :D

Reply:
Good catch :) But the duplication hurts me less, than thinking about a good name for those extracted files we could use both in h4 tag and the button :D If you have a good idea, go for it.

I think I wouldn't keep those styles in Typgraphy file, I'd remove this declaration from here and keep the styles in the button itself. : )